util/topgen.py

```diff
@@ -1036,11 +1038,6 @@ def main():
                         '-t',
                         required=True,
                         help="`top_{name}.hjson` file.")
-    parser.add_argument(
```

Comment:
Would it be useful to keep this (have it default to `util/topgen/templates`) for future 'tops' that may need different set of templates?